### DISCUSSION TITLE
Related products

### DISCUSSION THREAD

**Discussion by simonfrost** [duncanmcclean/simple-commerce#854] [Help]
I think we're close to choosing Simple Commerce for a new Statamic application, but wondered if there was an out of the box way to manage related products? Customer has a pretty fixed requirement for a 'related products / up-sells / cross-sells' feature on the product show view. All looks great by the way!

**Comment by duncanmcclean** [comment]
Hey!

There's not any functionality built-in to Simple Commerce to handle that. However, as your products are collections/entries, you'd do it the same way you would with any other type of content.

One potential way of approaching it would be to add tags/categories to your products and display related products based on them. However, I'm sure there are other ways to approach it.

**Reply by simonfrost** [reply]
Fair dos. Thanks for your quick response.
